docs/operational_guide/replication_and_deployment_in_zones.md

```diff
@@ -0,0 +1,57 @@
+# Replication and Deployment in Zones
+
+## Overview
+
+M3DB supports deployment to both a single zone, with rack level isolation, and deployment across multiple zones in a region. You can also deploy M3DB across multiple regions for a global view of data, however you may experience higher ingestion latency and cross region bandwidth costs. You may want to use a replication factor of five for such a multi-region cluster to survive multiple regions suffering an outage, or be able to still upgrade hosts while a region is down.
```

Comment:
> M3DB supports both deploying to a single zone with rack-level isolation and deploying across multiple zones in a region. It can also be deployed across multiple regions for a global view of data, though both ingestion latency and bandwidth costs may increase as a result. In multi-region clusters, a replication factor of five is recommended for upgrade and multi-region failure scenarios.

Reply:
TY, tweaked slightly but looks good.